src/routes/docs/quick-starts/web/+page.markdoc

```diff
@@ -0,0 +1,151 @@
+---
+layout: article
+title: Start with Web
+description: Build JavaScript or Typescript web apps with Appwrite and learn how to use our powerful backend to add authentication, user management, file storage, and more.
+difficulty: beginner
+readtime: 3
+back: /docs/quick-starts
+---
+
+Learn how to add Appwrite to your JavaScript or TypeScript web apps.
```

Comment:
This this is for the web I think javascript and typescript support can be assumed. How about: "Learn how to add Appwrite to your web apps."